src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,38 @@
+.page {
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: $font-size;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  width: $calendar-width;
+  padding: $calendar-padding;
+  gap: $gap;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.calendar__day {
+  box-sizing: border-box;
+  min-width: $width-cell;
+  min-height: $height-cell;
+  background-color: $background-cell;
+  border: 1px solid black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  transition:
+    transform 0.5s ease,
+    background-color 0.5s ease;
+}
+
+.calendar__day:hover {
+  cursor: pointer;
+  transform: translateY(-20px);
+  background-color: $background-cell-hover;
```

Comment:
The variable `$background-cell-hover` is used but not defined in this file. Ensure it is defined or imported from another SCSS file.